pages/use-solid.md

```diff
@@ -46,7 +46,7 @@ The following table lists some Pod Provider:
 | [redpencil.io](https://solid.redpencil.io/)                | [redpencil.io](https://redpencil.io/)                           | [Hetzner](https://www.hetzner.com/)        | Finland              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
 | [teamid.live](https://teamid.live/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)        | France              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
 
-If you are a provider who wants to add your service to this list, email [info@solidproject.org](mailto:info@solidproject.org).
+If you are a provider who wants to add your service to this list, please create a pull request to update [this file](https://github.com/solid/solidproject.org/blob/main/pages/use-solid.md).
```

Comment:
```suggestion
If you are a provider who wants to add your service to this list, please create a pull request to update [this file](https://github.com/solid/solidproject.org/blob/main/pages/use-solid.md). Alternatively, if you're unable to create a pull request, [open an Issue on this repository](https://github.com/solid/solidproject.org/issues/), and provide all the details someone else will need to create a new row for your service.
```

Reply:
I'd like this to be as self-serve as possible to limit the amount of work that needs to be done. So, I'd rather not provide the option to open an issue.

Reply:
Creating an Issue will remain an option for people who know GitHub, so the only people who will not know the option exists will be the people who are unable to create a PR, for whatever reason, keeping them off the list. I'm not in favor of this.

Reply:
The simplest assumption is that a "pod provider" would have the know-how to create a PR to include their service. So, no particular objection to offering issue as an alternative, but it is most likely unnecessary in this context.

@TallTed , do you have an objection to leaving out the issue option?

Reply:
I'll live with the omission.